initcpio/install/zfsbootmenu

```diff
@@ -184,6 +184,12 @@ build() {
         add_file "${_file}" "/libexec/teardown.d/${_file##*/}"
     done
 
+# shellcheck disable=SC2154
+    for _file in "${zfsbootmenu_load_key[@]}"; do
+        [ -x "${_file}" ] || continue
+        add_file "${_file}" "/libexec/load_key.d/${_file##*/}"
+    done
```

Comment:
Put this before the `zfsbootmenu_teardown` block just to keep the order logically consistent.